app/services/builders/metric_chart/blog/growth_month_over_month.rb

```diff
@@ -0,0 +1,26 @@
+module Builders
+  module MetricChart
+    module Blog
+      class GrowthMonthOverMonth < Builders::MetricChart::Blog::Base
+        def initialize(months = 13)
+          super(months + 1)
+        end
+
+        private
+
+        def process_data(metrics_data)
+          metrics_data.each_with_object({}) do |(month, metric_value), hash|
+            previous_metric_value = metrics_data[month.last_month]
+            next(hash) if previous_metric_value.nil? || previous_metric_value.zero?
+
+            hash[month] = growth_for(metric_value, previous_metric_value)
+          end
+        end
+
+        def growth_for(metric_value, previous_metric_value)
+          (metric_value - previous_metric_value) / previous_metric_value * 100
```

Comment:
I think this calc is wrong

```
[4] pry(main)> (3 - 2) / 2 * 100
=> 0
[5] pry(main)> (3 - 2) * 100 / 2
=> 50
```

Reply:
Great finding! I think it still works alright because the metric values extracted from the `metrics` table are floats, not integers. So the calculation would actually be like this:
```
[1] pry(main)> (3.0 - 2.0) / 2.0 * 100
 => 50.0 
```
Still, I'm gonna change it just in case

Reply:
oh good point 👍